docs/Guides/Porting.md

```diff
@@ -2,89 +2,63 @@
 
 You need to configure your user.email and user.name to check in code to the repositories:
 
 ```
 git config --global user.email "<Your E-Mail>"
 git config --global user.name "<Your Name>"
 ```
-This assumes that you have the latest version of [Git](https://my.rocketsoftware.com/RocketCommunity#/downloads) on your z/OS system.
+This assumes that you have the latest version of [Git](https://github.com/ZOSOpenTools/gitport/releases) on your z/OS system.
 
 
 ### Leveraging the z/OS Open Tools meta repo
 
-The meta repo (https://github.com/ZOSOpenTools/meta) consists of common tools and files that aid
-in the porting process, including the `zopen` suite of tools.  Specifically, `zopen build` provides a common way to bootstrap, configure, build, check,
-and install a software package.  `zopen download` provides a mechanism to download the latest published z/OS Open Tools.
+The meta repo (https://github.com/ZOSOpenTools/meta) consists of common tools and files that aid in the porting process, including the `zopen` suite of tools.  Specifically, `zopen build` provides a common way to bootstrap, configure, build, check,
+and install a software package. `zopen download` provides a mechanism to download the latest published z/OS Open Tools packages.
 
 Many tools depend on other tools to be able to build or run. You will need to provide both _bootstrap_ tools
 (i.e. binary tools not from source), as well as _prod_ tools (i.e. _production_ level tools previously built
 from another z/OS Open Tools repository).
 
-Our goal is to eventually have our own version of all the _bootstrap_ tools, but right now, we rely on some
-tools from Rocket. These Rocket tools can be downloaded [here](https://my.rocketsoftware.com/RocketCommunity#/downloads).
-
-Many tools require a C or C++ compiler (or both). 
-You can download a web deliverable add-on feature to your XL C/C++ compiler 
+Many tools require a C or C++ compiler (or both) to build. There a couple of options to obtain the C/C++ compiler:
+* You can download a web deliverable add-on feature to your XL C/C++ compiler 
 [here](https://www.ibm.com/servers/resourcelink/svc00100.nsf/pages/xlCC++V241ForZOsV24).
-Alternatively, you can install and manage _C/C++ for Open Enterprise Languages on z/OS_ using _RedHat OpenShift Container Platform_ and _IBM Z and Cloud Modernization Stack_ 
+* You can install and manage _C/C++ for Open Enterprise Languages on z/OS_ using _RedHat OpenShift Container Platform_ and _IBM Z and Cloud Modernization Stack_ 
 [here](https://github.com/IBM/z-and-cloud-modernization-stack-community). 
 Please note that these compilers are comparable, but how you perform installation and maintenance and pricing is different.
 
 In order for zopen to be able to locate dependent tools, they need to be in well-defined locations.
-Tools will be searched for in the following locations, in order:
+Tools will be searched for in the following default locations, in order:
```

Comment:
isn't 'dev' searched before prod?

Reply:
We don't search dev based on the code:
```
depsPath="$HOME/zopen/prod|$HOME/zopen/boot|/usr/bin/zopen/"
```
We'll probably change /usr/bin/zopen to /usr/local right? so I won't mention it until we do

Reply:
i agree not to mention /usr/bin/repo. Can you open up a separate issue for that? Also - if we don't search dev, i'm wondering if we should?